pandas/core/strings/accessor.py

```diff
@@ -1992,6 +1995,14 @@ def decode(self, encoding, errors: str = "strict"):
         ----------
         encoding : str
         errors : str, optional
+            Specifies the error handling scheme.
+            Possible values are those supported by :meth:`bytes.decode`.
```

Comment:
This doc created the conflict; not part of the origin PR I'm backporting but I figure why not add it.